packages/web3torrent/src/pages/file/File.test.tsx

```diff
@@ -9,18 +9,21 @@ import {WebTorrentAddInput} from '../../library/types';
 import {Status, Torrent} from '../../types';
 import {testSelector} from '../../utils/test-utils';
 import * as TorrentStatus from '../../utils/torrent-status-checker';
-import * as EmbeddedWalletClient from './../../clients/embedded-wallet-client';
 import * as Web3TorrentClient from './../../clients/web3torrent-client';
 import File from './File';
-import {JsonRpcResponse} from '@statechannels/channel-provider';
+
+// import {ChannelClient} from '@statechannels/channel-client';
+// import {JsonRpcResponse} from '@statechannels/channel-provider';
 
 const mockFileURL =
   '/file/#magnet:?xt=urn%3Abtih%3A148c62a7f7845c91e7d16ca9be85de6fbaed3a1f&dn=test.zip&xl=1398978&cost=0';
 
-const mockResponse: JsonRpcResponse = {jsonrpc: '2.0', id: 123, result: ''};
-
 Enzyme.configure({adapter: new Adapter()});
 
+// const mockResponse: JsonRpcResponse = {jsonrpc: '2.0', id: 123, result: ''};
```

Comment:
Are these dead comments needed? Can we clean them up?

Reply:
I only kept them because the commented out test relies on them.  Perhaps better to use `it.skip` instead and keep the code uncommented.